users/v1/users.proto

```diff
@@ -14,6 +15,11 @@ service Users{
   rpc LogoutUser(LogoutUserRequest) returns (LogoutUserResponse) {}
 }
 
+enum Status {
```

Comment:
what about others? like not found, bad request, validation error? I suggest unifying this. Something like dropping a special error message? We should discuss it.

Reply:
I mean, not found, bad request are kinda related to HTTP responses, so we shouldn't go strict in this way.